skopt/tests/test_space.py

```diff
@@ -84,6 +84,22 @@ def test_categorical_transform():
     assert_array_equal(ent_inverse, categories)
 
 
+def test_categorical_transform_binary():
+    categories = ["apple", "orange"]
+    cat = Categorical(categories)
+
+    apple = [0.]
+    orange = [1.]
+
+    assert_equal(cat.transform(["apple"]).size, cat.transformed_size)
```

Comment:
Can you just check this size explicilty, that is 

    `cat.transformed_size == 1`?